.phpunit.result.cache

```diff
@@ -0,0 +1 @@
":4,"Keboola\\Test\\Common\\BranchWorkspacesTest::testWorkspacePasswordReset":4,"Keboola\\Test\\Common\\BranchWorkspacesTest::testDropWorkspace with data set #0":4,"Keboola\\Test\\Common\\BranchWorkspacesTest::testDropNonExistingWorkspace with data set #0":4,"Keboola\\Test\\Common\\BranchWorkspacesTest::testDropNonExistingWorkspace with data set #1":4,"Keboola\\Test\\Options\\SearchTablesOptionsTest::testGetDefaults":4,"Keboola\\Test\\S3Uploader\\ChunkerTest::testSettersAndGetters":4,"Keboola\\Test\\S3Uploader\\PromiseResultHandlerTest::testGetRejected":4,"Keboola\\Test\\Common\\BranchWorkspacesTest::testWorkspaceCreate":4,"Keboola\\Test\\Common\\BranchWorkspacesTest::testDropWorkspace with data set #1":4},"times":{"Keboola\\Test\\Common\\BranchBucketsTest::testDropAllDevBucketsWhenDropBranch":0.021,"Keboola\\Test\\File\\AwsFileTest::testFileUpload with data set #0":0.013,"Keboola\\Test\\File\\AwsFileTest::testFileUpload with data set #2":0.019,"Keboola\\Test\\File\\AwsFileTest::testFileUpload with data set #3":0.012,"Keboola\\Test\\File\\AwsFileTest::testFileUpload with data set #5":0.025,"Keboola\\Test\\File\\AwsFileTest::testFileUploadUsingFederationToken with data set #0":0.013,"Keboola\\Test\\File\\AwsFileTest::testRequireEncryptionForSliced":0.028,"Keboola\\Test\\File\\AwsFileTest::testGetFileFederationToken":0.014,"Keboola\\Test\\Options\\FileUploadTransferOptionsTest::testSetters":0.023,"Keboola\\Test\\Options\\FileUploadTransferOptionsTest::testSetInvalidChunkSize":0.015,"Keboola\\Test\\Options\\FileUploadTransferOptionsTest::testSetInvalidMultiFileConcurrency":0.023,"Keboola\\Test\\Options\\FileUploadTransferOptionsTest::testSetInvalidSingleFileConcurrency":0.013,"Keboola\\Test\\Options\\SearchTablesOptionsTest::testCreate":0.017,"Keboola\\Test\\Options\\SearchTablesOptionsTest::testFluentInterface":0.016,"Keboola\\Test\\S3Uploader\\ChunkerTest::testSimpleChunker":0.016,"Keboola\\Test\\S3Uploader\\ChunkerTest::testAssociativeArrayChunker":0.018,"Keboola\\Test\\S3Uploader\\PromiseResultHandlerTest::testGetRejectedException":0.021,"Keboola\\Test\\Backend\\Workspaces\\ReadOnlyUserTest::testWorkspaceRestrictionsForReadOnlyUser":0.016,"Keboola\\Test\\Common\\BranchWorkspacesTest::testWorkspacePasswordReset":0.021,"Keboola\\Test\\Common\\BranchWorkspacesTest::testDropWorkspace with data set #0":0.021,"Keboola\\Test\\Common\\BranchWorkspacesTest::testDropNonExistingWorkspace with data set #0":0.013,"Keboola\\Test\\Common\\BranchWorkspacesTest::testDropNonExistingWorkspace with data set #1":0.012,"Keboola\\Test\\Downloader\\AbsUrlParserTest::testParseAbsUrlAzure":0.005,"Keboola\\Test\\Options\\SearchTablesOptionsTest::testGetDefaults":0.017,"Keboola\\Test\\S3Uploader\\ChunkerTest::testSettersAndGetters":0.018,"Keboola\\Test\\S3Uploader\\PromiseResultHandlerTest::testGetRejected":0.023,"Keboola\\Test\\Common\\BranchWorkspacesTest::testWorkspaceCreate":0.022,"Keboola\\Test\\Common\\BranchWorkspacesTest::testDropWorkspace with data set #1":0.017}}
```

Comment:
To mě přijde, že asi nemá byt commitnuté, ale v ignore, ne?

Reply:
Odebráno a ignorováno.